File: src/SextantTG/SextantTG.ActiveRecord/SubTour.cs
/*********************************************************
 * 
 *        Author: 郭旭
 * Last Modified: 2013-03-05
 *     File Name: SubTour.cs
 *   Description: 子旅行实体类。对于一次旅行而言，可能游览多个景点，
 *                每个景点的游览称为一个子旅行。
 * 
 * ******************************************************/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SextantTG.ActiveRecord
{
    /// <summary>
    /// 子旅行实体类
    /// </summary>
    public class SubTour
    {
        /// <summary>
        /// 所属旅行ID
        /// </summary>
        public string TourId { get; set; }
        
        /// <summary>
        /// 子旅行ID
        /// </summary>
        public string SubTourId { get; set; }

        /// <summary>
        /// 子旅行名称
        /// </summary>
        public string SubTourName { get; set; }

        /// <summary>
        /// 相关景点ID
        /// </summary>
        public string SightsId { get; set; }

        /// <summary>
        /// 起始时间
        /// </summary>
        public DateTime? BeginDate { get; set; }

        /// <summary>
        /// 结束时间
        /// </summary>
        public DateTime? EndDate { get; set; }

        /// <summary>
        /// 创建日期
        /// </summary>
        public DateTime? CreatingTime { get; set; }

        /// <summary>
        /// 备注
        /// </summary>
        public string Memos { get; set; }

    }
}

File: src/App/src/com/ss/stg/model/SubTour.java
package com.ss.stg.model;

import java.io.Serializable;
import java.util.Date;

/**
 * 子旅行
 * 
 * @author ggxx
 * 
 */
public class SubTour implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -4411878684644222925L;
	private String tourId;
	private String subTourId;
	private String subTourName;
	private String sightsId;
	private Date beginDate;
	private Date endDate;
	private Date creatingTime;
	private String memos;

	public SubTour() {
	}

	public String getTourId() {
		return tourId;
	}

	public void setTourId(String tourId) {
		this.tourId = tourId;
	}

	public String getSubTourId() {
		return subTourId;
	}

	public void setSubTourId(String subTourId) {
		this.subTourId = subTourId;
	}

	public String getSubTourName() {
		return subTourName;
	}

	public void setSubTourName(String subTourName) {
		this.subTourName = subTourName;
	}

	public String getSightsId() {
		return sightsId;
	}

	public void setSightsId(String sightsId) {
		this.sightsId = sightsId;
	}

	public Date getBeginDate() {
		return beginDate;
	}

	public void setBeginDate(Date beginDate) {
		this.beginDate = beginDate;
	}

	public Date getEndDate() {
		return endDate;
	}

	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}

	public Date getCreatingTime() {
		return creatingTime;
	}

	public void setCreatingTime(Date creatingTime) {
		this.creatingTime = creatingTime;
	}

	public String getMemos() {
		return memos;
	}

	public void setMemos(String memos) {
		this.memos = memos;
	}
}
